src/main/java/teammates/storage/entity/Notification.java

```diff
@@ -75,19 +78,22 @@ public Notification(Instant startTime, Instant endTime, String type, String targ
      * Instantiates a new notification, with all fields passed in as parameters.
      * This is mainly for conversion from attributes to entity.
      */
-    public Notification(String notificationId, Instant startTime, Instant endTime, String type, String targetUser,
-            String title, String message, boolean shown, Instant createdAt, Instant updatedAt) {
+    public Notification(String notificationId, Instant startTime, Instant endTime,
+                        NotificationType type, NotificationTargetUser targetUser,
```

Comment:
Perhaps we can leave this constructor alone? I think this can be used for PUT requests where we would not want createdAt to change, you could use the constructor above for a new notification object

Reply:
What do you mean by using this for PUT request? As I thought this is for creating a new notification instead of updating certain UpdateOptions